docs/data/toolpad/core/components/account/AccountCustomLocaleText.js

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { Account, AuthenticationContext, SessionContext } from '@toolpad/core';
```

Comment:
Here https://github.com/mui/toolpad/pull/4268 I'll be adding a rule not to import directly from `@toolpad/core, so we can do:

```tsx
import {
  AuthenticationContext,
  SessionContext,
} from '@toolpad/core/AppProvider';
import { Account } from '@toolpad/core/Account';
```